doc/code-submission-guidelines.md

```diff
@@ -23,11 +23,14 @@ Each commit message should consist of a header, a body and a footer. The header
 that includes a type, a scope and a subject. Here's a sample of the format:
 
 ```html
-<type>(<scope>): <subject>
-<BLANK LINE>
-<body>
-<BLANK LINE>
-<footer>
+<type
```

Comment:
Prettier seems to have done a very bad job here. We might need to manually format this one.

Reply:
This seems to be happening because we're treating this code block as HTML, but this code block is *not* HTML.

We should change this to a "raw" code fence (remove the `html` tag).